lib/AbsoluteGrid.jsx

```diff
@@ -60,6 +60,7 @@ export default class AbsoluteGrid extends React.Component {
       var gridItem = React.cloneElement(this.props.displayObject, _.assign(this.props.displayObject.props, {
         style: style,
         item: item,
+        itemsLength: this.props.items.length,
```

Comment:
i believe JS arrays maintain the `length` value as values are added/removed and do not compute it on each access, but I can make that change.